docs/stable/intro.md

```diff
@@ -25,6 +29,7 @@ ServerlessLLM is a **fast** and **easy-to-use** serving system designed for **af
 ### ServerlessLLM Store
 
 - [Quickstart](./store/quickstart.md)
+- [ROCm Installation(Experimental)](./store/installation_with_rocm.md)
```

Comment:
@SiyangShao Do you think it would be better to move the [Tested Hardware](https://serverlessllm.github.io/docs/stable/store/installation_with_rocm#tested-hardware) section to the beginning of this document?

Reply:
Sure